resources/normal/base/lang/tintin.pot

```diff
@@ -0,0 +1,2836 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
we may need some real data here?

Reply:
we can figure this out later.  filed #229